portal/views/organization.py

```diff
@@ -496,8 +508,34 @@ def add_user_organizations(user_id):
     if not request.json or 'organizations' not in request.json:
         abort(400, "Requires `organizations` list")
 
+    # validate input - don't allow multiple orgs with `apply_to_user`
+    # attributes for any given field
+    applied_fields = []
+
+    def apply_defaults(item, org, applied):
+        for field in 'timezone', 'language':
+            if field in item:
```

Comment:
I might invert this condition and `continue` to de-indent the meat of the code a level